core-bundle/src/Resources/contao/templates/_new/component/_headline.html.twig

```diff
@@ -0,0 +1,11 @@
+<{{ headline.tagName }}>
+{%- block inner %}
+    {%- apply spaceless %}
+        {% if as_overview %}
```

Comment:
This `as_overview` is really strange. Overview of what? I found "back end wildcard" a lot better. Not because I knew what a "wildcard" was but it had a name and by learning what it is, we understood what is meant. "overview" is just a general term...

Reply:
IIRC @ausi and I came up with `as_overview` as a way to tell the different "output scopes" apart. Having something like backend_preview/wildcard/overview… would to me imply that this can only work in the back end. Can we maybe find a better term that does not include back end? :slightly_smiling_face: 

`as_summary`? 

I personally like the `as_` prefix, but open for suggestions of course!

Reply:
Naming...so let me try to articulate what this does: We decide between what should be done when actually executing the template (FE) and the "raw" view for the editor in the back end (I agree back end is not good because you might want to render it in FE mode in the BE).
What do you think about `as_editor_view`? It's the view for the one that knows what insert tags etc. are, right?

Reply:
I like `as_editor_view`!

Reply:
+1 for `as_editor_view` ❤️

Reply:
I would like to throw in `is_preview`.

Reply:
I went with `as_editor_view` in 1448234be35e3959b83a530868f1aa56bcff1bfd for now,

Reply:
My reasoning was output this "as editor view" i.e. a short form of `shouldBeOutputAsEditorView` instead of `isInEditorView` but ultimately I'd be fine with both.

Reply:
I’m fine with both too (`as_editor_view`, `is_editor_view`).

Reply:
Same here. I don't care. My problem is the term "overview". I agree with @m-vo that you would say "render as" but also, it's not typical in a template variable :)